### ISSUE TITLE
ui: not found page can't be hit when authenticated

### ISSUE THREAD

**Issue by wass3rw3rk** [go-vela/community#973]
## Description

currently, the not found page can be hit when unauthenticated, once you are authenticated it can not - instead you are always redirected to "/".

## Useful Information

affects currently unreleased UI

1. What is the output of `vela --version`?
v0.25 (unreleased)

3. What operating system is being used?

4. Any other important details?

**Comment by plyr4** [comment]
was looking into this and would love to build out a better 404 experience but im not sure what it looks like.

in a perfect world, what _is_ the not found page when you are authenticated and have dynamic routes for `vela.com/<org>/<repo>` ?

**Comment by wass3r** [comment]
yes, for sure. we can get creative. i know we talked about `vela.com/<org>/<repo>` before. i imagine something like "looks like this repo is not setup yet. do you want to enable it?" kind of messaging.

any org level page that doesn't have any repos enabled, probably something in a similar vein. maybe prompt if the user wants to enable a repo within that org? perhaps, offer the ability to browse repos filtered to that org that they have access to?

so, i think 404/not found perhaps best adjusts to each scenario, but there should be a generic one that's just, umm.. a 404-themed mini-game and you have to beat to get back to the home/overview page? :D
